bundles/org.openhab.core/src/main/java/org/openhab/core/internal/items/MetadataStateDescriptionFragmentProvider.java

```diff
@@ -83,10 +83,10 @@ public MetadataStateDescriptionFragmentProvider(final @Reference MetadataRegistr
                     builder.withMinimum(getBigDecimal(metadata.getConfiguration().get("min")));
                 }
                 if (metadata.getConfiguration().containsKey("max")) {
-                    builder.withMaximum(getBigDecimal(metadata.getConfiguration().get("min")));
+                    builder.withMaximum(getBigDecimal(metadata.getConfiguration().get("max")));
```

Comment:
haha, it seems the unit tests have already helped ;-)

Reply:
My thoughts exactly but I wanted to keep that under the radar ;)